src/doc/config.md

```diff
@@ -73,4 +73,8 @@ token = "..."   # Access token (found on the central repo's website)
 
 [http]
 proxy = "..."   # HTTP proxy to use for HTTP requests (defaults to none)
+timeout = 30    # Timeout for all HTTP requests, in seconds
```

Comment:
[Currently, yes](https://github.com/carllerche/curl-rust/blob/master/src/http/handle.rs#L15)

Although now that I realize it's in milliseconds, I'll change the documentation here!